Help running 2022 NeurIPS Competition agents

### High Level Description

Hello, I am trying to run a benchmark on the 2022 NeurIPS Competition agents which were added to SMARTS. I followed all the instructions from the [ReadTheDocs page](https://smarts.readthedocs.io/en/latest/benchmarks/driving_smarts_2022.html) and executed the following steps: 

`cd <path>/SMARTS`
`scl zoo install zoo/policies/interaction_aware_motion_prediction`
`scl benchmark run driving_smarts==0.0 zoo.policies:interaction-aware-motion-prediction-agent-v0 --auto-install`

The installation comletes successfully but when runnung the benchmark I get the following error:

<pre>(.venv) <font color="#26A269"><b>atanas@virtualbox</b></font>:<font color="#12488B"><b>~/SMARTS</b></font>$ scl benchmark run driving_smarts==0.0 zoo.policies:interaction-aware-motion-prediction-agent-v0 --auto-install
Requirement already satisfied: ray&lt;=2.2.0,&gt;2.0 in ./.venv/lib/python3.8/site-packages (2.2.0)
Requirement already satisfied: numpy&gt;=1.16 in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (1.23.1)
Requirement already satisfied: click&gt;=7.0 in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (8.1.3)
Requirement already satisfied: aiosignal in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (1.3.1)
Requirement already satisfied: grpcio&gt;=1.32.0 in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (1.53.0)
Requirement already satisfied: jsonschema in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (4.17.3)
Requirement already satisfied: requests in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (2.28.2)
Requirement already satisfied: frozenlist in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (1.3.3)
Requirement already satisfied: virtualenv&gt;=20.0.24 in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (20.21.0)
Requirement already satisfied: attrs in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (22.2.0)
Requirement already satisfied: protobuf!=3.19.5,&gt;=3.15.3 in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (3.20.1)
Requirement already satisfied: filelock in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (3.10.7)
Requirement already satisfied: msgpack&lt;2.0.0,&gt;=1.0.0 in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (1.0.5)
Requirement already satisfied: pyyaml in ./.venv/lib/python3.8/site-packages (from ray&lt;=2.2.0,&gt;2.0) (6.0)
Requirement already satisfied: platformdirs&lt;4,&gt;=2.4 in ./.venv/lib/python3.8/site-packages (from virtualenv&gt;=20.0.24-&gt;ray&lt;=2.2.0,&gt;2.0) (3.2.0)
Requirement already satisfied: distlib&lt;1,&gt;=0.3.6 in ./.venv/lib/python3.8/site-packages (from virtualenv&gt;=20.0.24-&gt;ray&lt;=2.2.0,&gt;2.0) (0.3.6)
Requirement already satisfied: importlib-resources&gt;=1.4.0 in ./.venv/lib/python3.8/site-packages (from jsonschema-&gt;ray&lt;=2.2.0,&gt;2.0) (5.12.0)
Requirement already satisfied: pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,&gt;=0.14.0 in ./.venv/lib/python3.8/site-packages (from jsonschema-&gt;ray&lt;=2.2.0,&gt;2.0) (0.19.3)
Requirement already satisfied: pkgutil-resolve-name&gt;=1.3.10 in ./.venv/lib/python3.8/site-packages (from jsonschema-&gt;ray&lt;=2.2.0,&gt;2.0) (1.3.10)
Requirement already satisfied: charset-normalizer&lt;4,&gt;=2 in ./.venv/lib/python3.8/site-packages (from requests-&gt;ray&lt;=2.2.0,&gt;2.0) (3.1.0)
Requirement already satisfied: idna&lt;4,&gt;=2.5 in ./.venv/lib/python3.8/site-packages (from requests-&gt;ray&lt;=2.2.0,&gt;2.0) (3.4)
Requirement already satisfied: urllib3&lt;1.27,&gt;=1.21.1 in ./.venv/lib/python3.8/site-packages (from requests-&gt;ray&lt;=2.2.0,&gt;2.0) (1.26.15)
Requirement already satisfied: certifi&gt;=2017.4.17 in ./.venv/lib/python3.8/site-packages (from requests-&gt;ray&lt;=2.2.0,&gt;2.0) (2022.12.7)
Requirement already satisfied: zipp&gt;=3.1.0 in ./.venv/lib/python3.8/site-packages (from importlib-resources&gt;=1.4.0-&gt;jsonschema-&gt;ray&lt;=2.2.0,&gt;2.0) (3.15.0)
Traceback (most recent call last):
  File &quot;/home/atanas/SMARTS/.venv/bin/scl&quot;, line 8, in &lt;module&gt;
    sys.exit(scl())
  File &quot;/home/atanas/SMARTS/.venv/lib/python3.8/site-packages/click/core.py&quot;, line 1130, in __call__
    return self.main(*args, **kwargs)
  File &quot;/home/atanas/SMARTS/.venv/lib/python3.8/site-packages/click/core.py&quot;, line 1055, in main
    rv = self.invoke(ctx)
  File &quot;/home/atanas/SMARTS/.venv/lib/python3.8/site-packages/click/core.py&quot;, line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File &quot;/home/atanas/SMARTS/.venv/lib/python3.8/site-packages/click/core.py&quot;, line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File &quot;/home/atanas/SMARTS/.venv/lib/python3.8/site-packages/click/core.py&quot;, line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File &quot;/home/atanas/SMARTS/.venv/lib/python3.8/site-packages/click/core.py&quot;, line 760, in invoke
    return __callback(*args, **kwargs)
  File &quot;/home/atanas/SMARTS/.venv/lib/python3.8/site-packages/cli/benchmark.py&quot;, line 79, in run
    run_benchmark(
  File &quot;/home/atanas/SMARTS/.venv/lib/python3.8/site-packages/smarts/benchmark/__init__.py&quot;, line 109, in run_benchmark
    entrypoint(
  File &quot;/home/atanas/SMARTS/.venv/lib/python3.8/site-packages/smarts/benchmark/entrypoints/benchmark_runner_v0.py&quot;, line 204, in benchmark_from_configs
    assert agent_args, f&quot;&quot;&quot;
AssertionError: 
    Cannot resolve `zoo.policies:interaction-aware-motion-prediction-agent-v0`. This should be in a location that can be resolved by python
    in python&apos;s `sys.path`.  (e.g. `&lt;path&gt;/custom/__init__.py` or`&lt;path&gt;/custom.py`) 
    
    Please ensure agent configuration:
      - file exists
      - file path is correct
      - contains correct data

    The benchmark cannot continue.
</pre>

How can I resolve this?
Thank you


### Version

SMARTS 1.0.9 (latest)

### Operating System

Ubuntu 20.04

### Problems

_No response_

--- a/zoo/policies/interaction_aware_motion_prediction/setup.cfg
+++ b/zoo/policies/interaction_aware_motion_prediction/setup.cfg
@@ -1,6 +1,6 @@
 [metadata]
 name = smarts_agent_iamp_stub
-version = 0.1.0
+version = 0.1.1
 url = https://github.com/huawei-noah/SMARTS
 description = SMARTS zoo agent.
 long_description = Trained Interaction-aware Motion Prediction agent. See [SMARTS](https://github.com/huawei-noah/SMARTS).
@@ -16,4 +16,4 @@ include_package_data = True
 zip_safe = True
 python_requires = == 3.8.*
 install_requires = 
-    smarts_agent_iamp==0.1.0
+    smarts_agent_iamp==0.1.1

